visivo/parsers/mkdocs_utils/markdown.py

```diff
@@ -0,0 +1,154 @@
+from textwrap import dedent
+import yaml
+
+
+def find_refs(obj):
```

Comment:
Not sure if there is an opportunity to use the models themselves and the dag to do this work, since we do some of this kind of searching in other places.

Reply:
My naming convention might be a little misleading. It's trying to find models that are referenced somewhere in the properties rather than replace the ref strings with models.